src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
     var picture = playerObj.stateObj.attributes.entity_picture;
+    var dummy;
     if (picture) {
-      this.$.cover.style.backgroundImage = 'url(' + picture + ')';
+      dummy = document.createElement('IMG');
```

Comment:
The advantage would be that we aren't creating an element and registering events on every state change. I also think that as a high-level idea that making use of the polymer features when available will be more stable than writing our own javascript implementations. This instance should be pretty safe, but using polymer as a practice lets take advantage of their compatibility layer for various browsers.

It will probably be a little larger in LOC, but IMO moving the load/error handlers outside of the playerObjChanged handler will make it easier to quickly grok.